pkg/booleanpolicy/field_metadata.go

```diff
@@ -888,16 +897,18 @@ func initializeFieldMetadata() FieldMetadata {
 		[]RuntimeFieldType{}, operatorsForbidden,
 	)
 
-	f.registerFieldMetadata(fieldnames.FilePath,
-		querybuilders.ForFieldLabel(search.FilePath), nil,
-		func(*validateConfiguration) *regexp.Regexp {
-			// TODO(ROX-31449): change to an absolute path regex when arbitrary
-			// paths are supported
-			return allowedFilePathRegex
-		},
-		[]storage.EventSource{storage.EventSource_NODE_EVENT},
-		[]RuntimeFieldType{FileAccess}, negationForbidden,
-	)
+	if features.SensitiveFileActivity.Enabled() {
+		f.registerFieldMetadataConditionally(fieldnames.FilePath,
```

Comment:
With my [PR](https://github.com/stackrox/stackrox/pull/17127) merging this will have to be `registerFieldMetadata`

Reply:
woops, I forgot to change this back when I was fiddling